{
  "0e03a480debcb54dd0ad4bed14f612d2" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 10,
      "skipped" : 0,
      "passed" : 13,
      "unknown" : 0,
      "total" : 23
    },
    "items" : [ {
      "uid" : "44ca7c89755b90c4",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/10//#testresult/44ca7c89755b90c4",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6b14fca860>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649166993086,
        "stop" : 1649166993086,
        "duration" : 0
      }
    }, {
      "uid" : "b7169c05b242a5ef",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/9//#testresult/b7169c05b242a5ef",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f04e8582680>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649166018410,
        "stop" : 1649166018410,
        "duration" : 0
      }
    }, {
      "uid" : "b99cf69d7d6efac7",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/20//#testresult/b99cf69d7d6efac7",
      "status" : "passed",
      "time" : {
        "start" : 1649165665784,
        "stop" : 1649165666438,
        "duration" : 654
      }
    }, {
      "uid" : "d48ef0ca2afb7bd6",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/19//#testresult/d48ef0ca2afb7bd6",
      "status" : "passed",
      "time" : {
        "start" : 1649165604776,
        "stop" : 1649165605600,
        "duration" : 824
      }
    }, {
      "uid" : "44264753d782984",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/7//#testresult/44264753d782984",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f69c035a5f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649165599608,
        "stop" : 1649165599608,
        "duration" : 0
      }
    }, {
      "uid" : "c2f1ec86a1090425",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/6//#testresult/c2f1ec86a1090425",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe68076be50>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649110750016,
        "stop" : 1649110750016,
        "duration" : 0
      }
    }, {
      "uid" : "2597cc5507ee15eb",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/18//#testresult/2597cc5507ee15eb",
      "status" : "passed",
      "time" : {
        "start" : 1649110731073,
        "stop" : 1649110731815,
        "duration" : 742
      }
    }, {
      "uid" : "bb45db284b347903",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/17//#testresult/bb45db284b347903",
      "status" : "passed",
      "time" : {
        "start" : 1649109323001,
        "stop" : 1649109323755,
        "duration" : 754
      }
    }, {
      "uid" : "a49614a343ed6414",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/4//#testresult/a49614a343ed6414",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff12e50a2c0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649109176418,
        "stop" : 1649109176418,
        "duration" : 0
      }
    }, {
      "uid" : "e7fc2e1c90d00444",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/3//#testresult/e7fc2e1c90d00444",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f565220a290>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649109036771,
        "stop" : 1649109036771,
        "duration" : 0
      }
    }, {
      "uid" : "1c57645aa77ff170",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/14//#testresult/1c57645aa77ff170",
      "status" : "passed",
      "time" : {
        "start" : 1649108805617,
        "stop" : 1649108806460,
        "duration" : 843
      }
    }, {
      "uid" : "12198787a79adf84",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/13//#testresult/12198787a79adf84",
      "status" : "passed",
      "time" : {
        "start" : 1649108546412,
        "stop" : 1649108547347,
        "duration" : 935
      }
    }, {
      "uid" : "b3f0881205416860",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/1//#testresult/b3f0881205416860",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa14dcffd30>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649108545068,
        "stop" : 1649108545068,
        "duration" : 0
      }
    }, {
      "uid" : "9b7039273e569dcf",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/11//#testresult/9b7039273e569dcf",
      "status" : "passed",
      "time" : {
        "start" : 1649108211796,
        "stop" : 1649108212397,
        "duration" : 601
      }
    }, {
      "uid" : "8ca81a019efdc452",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/10//#testresult/8ca81a019efdc452",
      "status" : "passed",
      "time" : {
        "start" : 1649107865819,
        "stop" : 1649107866508,
        "duration" : 689
      }
    }, {
      "uid" : "67b1be647300e6cb",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/9//#testresult/67b1be647300e6cb",
      "status" : "passed",
      "time" : {
        "start" : 1649107516146,
        "stop" : 1649107516917,
        "duration" : 771
      }
    }, {
      "uid" : "3b99115c2c81a078",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/8//#testresult/3b99115c2c81a078",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x560523969ad3 <unknown>\n#1 0x5605236c9568 <unknown>\n#2 0x5605236ecb37 <unknown>\n#3 0x5605236e823a <unknown>\n#4 0x5605237232ba <unknown>\n#5 0x56052371d4e3 <unknown>\n#6 0x5605236f2d1a <unknown>\n#7 0x5605236f3e75 <unknown>\n#8 0x560523997efd <unknown>\n#9 0x5605239b119b <unknown>\n#10 0x560523999c65 <unknown>\n#11 0x5605239b1ec8 <unknown>\n#12 0x56052398d360 <unknown>\n#13 0x5605239cda68 <unknown>\n#14 0x5605239cdbe8 <unknown>\n#15 0x5605239e77fd <unknown>\n#16 0x7f277e044609 <unknown>",
      "time" : {
        "start" : 1649107302485,
        "stop" : 1649107302485,
        "duration" : 0
      }
    }, {
      "uid" : "a10e1e421c6ce67e",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/7//#testresult/a10e1e421c6ce67e",
      "status" : "passed",
      "time" : {
        "start" : 1649106453830,
        "stop" : 1649106454645,
        "duration" : 815
      }
    }, {
      "uid" : "519535eb4dc9a376",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/6//#testresult/519535eb4dc9a376",
      "status" : "passed",
      "time" : {
        "start" : 1649106438862,
        "stop" : 1649106439505,
        "duration" : 643
      }
    }, {
      "uid" : "2899f859d763f88e",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/5//#testresult/2899f859d763f88e",
      "status" : "passed",
      "time" : {
        "start" : 1649106385753,
        "stop" : 1649106386612,
        "duration" : 859
      }
    } ]
  },
  "4c6a26788916d97b0c591def5d62e104" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 7,
      "skipped" : 0,
      "passed" : 11,
      "unknown" : 0,
      "total" : 18
    },
    "items" : [ {
      "uid" : "6d9a128ba38d75e9",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/6//#testresult/6d9a128ba38d75e9",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe67e44e1a0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649110750407,
        "stop" : 1649110750407,
        "duration" : 0
      }
    }, {
      "uid" : "fb93cba2a679fc12",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/18//#testresult/fb93cba2a679fc12",
      "status" : "passed",
      "time" : {
        "start" : 1649110733766,
        "stop" : 1649110734391,
        "duration" : 625
      }
    }, {
      "uid" : "27b30639d46f7642",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/17//#testresult/27b30639d46f7642",
      "status" : "passed",
      "time" : {
        "start" : 1649109325988,
        "stop" : 1649109326601,
        "duration" : 613
      }
    }, {
      "uid" : "e557ccd94148afb",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/4//#testresult/e557ccd94148afb",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff12e4de1a0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649109177079,
        "stop" : 1649109177079,
        "duration" : 0
      }
    }, {
      "uid" : "dc76ae02554d0aca",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/3//#testresult/dc76ae02554d0aca",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f56521da200>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649109037204,
        "stop" : 1649109037204,
        "duration" : 0
      }
    }, {
      "uid" : "e7343f326b7fdc6c",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/14//#testresult/e7343f326b7fdc6c",
      "status" : "passed",
      "time" : {
        "start" : 1649108808828,
        "stop" : 1649108809575,
        "duration" : 747
      }
    }, {
      "uid" : "908846b93301592a",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/13//#testresult/908846b93301592a",
      "status" : "passed",
      "time" : {
        "start" : 1649108549718,
        "stop" : 1649108550489,
        "duration" : 771
      }
    }, {
      "uid" : "1fc1fcc387eb4cbe",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/1//#testresult/1fc1fcc387eb4cbe",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa14c8c61d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649108545433,
        "stop" : 1649108545433,
        "duration" : 0
      }
    }, {
      "uid" : "2ee38d395c1dd3a1",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/11//#testresult/2ee38d395c1dd3a1",
      "status" : "passed",
      "time" : {
        "start" : 1649108214302,
        "stop" : 1649108217073,
        "duration" : 2771
      }
    }, {
      "uid" : "eec675ba93b865fd",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/10//#testresult/eec675ba93b865fd",
      "status" : "passed",
      "time" : {
        "start" : 1649107868330,
        "stop" : 1649107869041,
        "duration" : 711
      }
    }, {
      "uid" : "91ef0422c52c080",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/9//#testresult/91ef0422c52c080",
      "status" : "passed",
      "time" : {
        "start" : 1649107519146,
        "stop" : 1649107519892,
        "duration" : 746
      }
    }, {
      "uid" : "c49c1518bec25704",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/8//#testresult/c49c1518bec25704",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x56191941dad3 <unknown>\n#1 0x56191917d568 <unknown>\n#2 0x5619191a0b37 <unknown>\n#3 0x56191919c23a <unknown>\n#4 0x5619191d72ba <unknown>\n#5 0x5619191d14e3 <unknown>\n#6 0x5619191a6d1a <unknown>\n#7 0x5619191a7e75 <unknown>\n#8 0x56191944befd <unknown>\n#9 0x56191946519b <unknown>\n#10 0x56191944dc65 <unknown>\n#11 0x561919465ec8 <unknown>\n#12 0x561919441360 <unknown>\n#13 0x561919481a68 <unknown>\n#14 0x561919481be8 <unknown>\n#15 0x56191949b7fd <unknown>\n#16 0x7f1dfa3f4609 <unknown>",
      "time" : {
        "start" : 1649107303973,
        "stop" : 1649107303973,
        "duration" : 0
      }
    }, {
      "uid" : "562eecd077a01d",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/7//#testresult/562eecd077a01d",
      "status" : "passed",
      "time" : {
        "start" : 1649106456854,
        "stop" : 1649106457689,
        "duration" : 835
      }
    }, {
      "uid" : "25a04c666a9191d0",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/6//#testresult/25a04c666a9191d0",
      "status" : "passed",
      "time" : {
        "start" : 1649106441517,
        "stop" : 1649106442118,
        "duration" : 601
      }
    }, {
      "uid" : "5e378b6c60343aba",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/5//#testresult/5e378b6c60343aba",
      "status" : "passed",
      "time" : {
        "start" : 1649106389347,
        "stop" : 1649106390143,
        "duration" : 796
      }
    }, {
      "uid" : "e7dd04b518473d21",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/4//#testresult/e7dd04b518473d21",
      "status" : "passed",
      "time" : {
        "start" : 1649105941257,
        "stop" : 1649105941913,
        "duration" : 656
      }
    }, {
      "uid" : "b8d27bdb16a7d0aa",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/3//#testresult/b8d27bdb16a7d0aa",
      "status" : "broken",
      "statusDetails" : "NameError: name 'webdriver' is not defined",
      "time" : {
        "start" : 1649105743283,
        "stop" : 1649105743283,
        "duration" : 0
      }
    }, {
      "uid" : "1145381f9c1dd9f5",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/2//#testresult/1145381f9c1dd9f5",
      "status" : "broken",
      "statusDetails" : "ValueError: no option named '--host'",
      "time" : {
        "start" : 1649105573035,
        "stop" : 1649105573035,
        "duration" : 0
      }
    } ]
  },
  "b2e57dac307fb5f5a232440dbae3faf9" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 10,
      "skipped" : 0,
      "passed" : 13,
      "unknown" : 0,
      "total" : 23
    },
    "items" : [ {
      "uid" : "597602b233a670ff",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/10//#testresult/597602b233a670ff",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6b1269f0d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649166993384,
        "stop" : 1649166993384,
        "duration" : 0
      }
    }, {
      "uid" : "eb23e78f8c357246",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/9//#testresult/eb23e78f8c357246",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f04e8306fb0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649166018724,
        "stop" : 1649166018724,
        "duration" : 0
      }
    }, {
      "uid" : "a42ce630d08e7e9a",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/20//#testresult/a42ce630d08e7e9a",
      "status" : "passed",
      "time" : {
        "start" : 1649165667142,
        "stop" : 1649165667756,
        "duration" : 614
      }
    }, {
      "uid" : "e87c4f3c4cd0a787",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/19//#testresult/e87c4f3c4cd0a787",
      "status" : "passed",
      "time" : {
        "start" : 1649165606391,
        "stop" : 1649165607180,
        "duration" : 789
      }
    }, {
      "uid" : "797e8845e1eb9d18",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/7//#testresult/797e8845e1eb9d18",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f69bdb96140>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649165599849,
        "stop" : 1649165599849,
        "duration" : 0
      }
    }, {
      "uid" : "da9698e7d252f701",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/6//#testresult/da9698e7d252f701",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe67e2e9b70>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649110750171,
        "stop" : 1649110750171,
        "duration" : 0
      }
    }, {
      "uid" : "d70bfedc64cc96aa",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/18//#testresult/d70bfedc64cc96aa",
      "status" : "passed",
      "time" : {
        "start" : 1649110732518,
        "stop" : 1649110733136,
        "duration" : 618
      }
    }, {
      "uid" : "1ed5ab82131a62a8",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/17//#testresult/1ed5ab82131a62a8",
      "status" : "passed",
      "time" : {
        "start" : 1649109324543,
        "stop" : 1649109325214,
        "duration" : 671
      }
    }, {
      "uid" : "686ef213f880f490",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/4//#testresult/686ef213f880f490",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff130a65c00>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649109176744,
        "stop" : 1649109176744,
        "duration" : 0
      }
    }, {
      "uid" : "a2add1e3ce4238fc",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/3//#testresult/a2add1e3ce4238fc",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5652119b70>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649109036992,
        "stop" : 1649109036992,
        "duration" : 0
      }
    }, {
      "uid" : "e4a9c8f1fecf7a2d",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/14//#testresult/e4a9c8f1fecf7a2d",
      "status" : "passed",
      "time" : {
        "start" : 1649108807276,
        "stop" : 1649108808138,
        "duration" : 862
      }
    }, {
      "uid" : "d9a39cdf8658e06b",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/13//#testresult/d9a39cdf8658e06b",
      "status" : "passed",
      "time" : {
        "start" : 1649108548138,
        "stop" : 1649108548882,
        "duration" : 744
      }
    }, {
      "uid" : "e135502b6f2567c",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/1//#testresult/e135502b6f2567c",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa14eda1bd0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649108545242,
        "stop" : 1649108545242,
        "duration" : 0
      }
    }, {
      "uid" : "b877a978ea236190",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/11//#testresult/b877a978ea236190",
      "status" : "passed",
      "time" : {
        "start" : 1649108212993,
        "stop" : 1649108213714,
        "duration" : 721
      }
    }, {
      "uid" : "ae95e3c4384d217f",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/10//#testresult/ae95e3c4384d217f",
      "status" : "passed",
      "time" : {
        "start" : 1649107867070,
        "stop" : 1649107867710,
        "duration" : 640
      }
    }, {
      "uid" : "b61390b1999cb316",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/9//#testresult/b61390b1999cb316",
      "status" : "passed",
      "time" : {
        "start" : 1649107517690,
        "stop" : 1649107518405,
        "duration" : 715
      }
    }, {
      "uid" : "c1be31e5e765426",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/8//#testresult/c1be31e5e765426",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x56466e658ad3 <unknown>\n#1 0x56466e3b8568 <unknown>\n#2 0x56466e3dbb37 <unknown>\n#3 0x56466e3d723a <unknown>\n#4 0x56466e4122ba <unknown>\n#5 0x56466e40c4e3 <unknown>\n#6 0x56466e3e1d1a <unknown>\n#7 0x56466e3e2e75 <unknown>\n#8 0x56466e686efd <unknown>\n#9 0x56466e6a019b <unknown>\n#10 0x56466e688c65 <unknown>\n#11 0x56466e6a0ec8 <unknown>\n#12 0x56466e67c360 <unknown>\n#13 0x56466e6bca68 <unknown>\n#14 0x56466e6bcbe8 <unknown>\n#15 0x56466e6d67fd <unknown>\n#16 0x7fe28fe92609 <unknown>",
      "time" : {
        "start" : 1649107303228,
        "stop" : 1649107303228,
        "duration" : 0
      }
    }, {
      "uid" : "11fd7c29fdaadffd",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/7//#testresult/11fd7c29fdaadffd",
      "status" : "passed",
      "time" : {
        "start" : 1649106455329,
        "stop" : 1649106456115,
        "duration" : 786
      }
    }, {
      "uid" : "a64cee9db9906ff8",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/6//#testresult/a64cee9db9906ff8",
      "status" : "passed",
      "time" : {
        "start" : 1649106440114,
        "stop" : 1649106440812,
        "duration" : 698
      }
    }, {
      "uid" : "c097bb6db4eafbc5",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/5//#testresult/c097bb6db4eafbc5",
      "status" : "passed",
      "time" : {
        "start" : 1649106387398,
        "stop" : 1649106388604,
        "duration" : 1206
      }
    } ]
  },
  "65fd2b4dfcf9c1e5ba26298719a99a88" : {
    "statistic" : {
      "failed" : 2,
      "broken" : 10,
      "skipped" : 0,
      "passed" : 11,
      "unknown" : 0,
      "total" : 23
    },
    "items" : [ {
      "uid" : "ea2244b7a9feaa26",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/10//#testresult/ea2244b7a9feaa26",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6b12635b70>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649166993570,
        "stop" : 1649166993570,
        "duration" : 0
      }
    }, {
      "uid" : "257170fd0a3a80cf",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/9//#testresult/257170fd0a3a80cf",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f04e82d5de0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649166019066,
        "stop" : 1649166019066,
        "duration" : 0
      }
    }, {
      "uid" : "1d80904073a735cc",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/20//#testresult/1d80904073a735cc",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert False",
      "time" : {
        "start" : 1649165668388,
        "stop" : 1649165669100,
        "duration" : 712
      }
    }, {
      "uid" : "22cc18f00a1cf0ae",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/19//#testresult/22cc18f00a1cf0ae",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert False",
      "time" : {
        "start" : 1649165607884,
        "stop" : 1649165608696,
        "duration" : 812
      }
    }, {
      "uid" : "3864d48949a32a4b",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/7//#testresult/3864d48949a32a4b",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f69c00f5c00>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649165600036,
        "stop" : 1649165600036,
        "duration" : 0
      }
    }, {
      "uid" : "2df502601f9bbcec",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/6//#testresult/2df502601f9bbcec",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe6809ec8b0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649110749784,
        "stop" : 1649110749784,
        "duration" : 0
      }
    }, {
      "uid" : "a811dbc725738448",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/18//#testresult/a811dbc725738448",
      "status" : "passed",
      "time" : {
        "start" : 1649110729602,
        "stop" : 1649110730403,
        "duration" : 801
      }
    }, {
      "uid" : "4ebfbca5d49bb805",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/17//#testresult/4ebfbca5d49bb805",
      "status" : "passed",
      "time" : {
        "start" : 1649109321731,
        "stop" : 1649109322377,
        "duration" : 646
      }
    }, {
      "uid" : "478735d34d80760",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/4//#testresult/478735d34d80760",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff130cd0820>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649109176032,
        "stop" : 1649109176032,
        "duration" : 0
      }
    }, {
      "uid" : "10765b1d788039fc",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/3//#testresult/10765b1d788039fc",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f56549cc850>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649109036493,
        "stop" : 1649109036493,
        "duration" : 0
      }
    }, {
      "uid" : "3e0125074b178a3a",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/14//#testresult/3e0125074b178a3a",
      "status" : "passed",
      "time" : {
        "start" : 1649108803930,
        "stop" : 1649108804774,
        "duration" : 844
      }
    }, {
      "uid" : "483ba636da7627bf",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/13//#testresult/483ba636da7627bf",
      "status" : "passed",
      "time" : {
        "start" : 1649108544889,
        "stop" : 1649108545644,
        "duration" : 755
      }
    }, {
      "uid" : "9f9b4a8cba06bec",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/1//#testresult/9f9b4a8cba06bec",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa14f0008b0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "time" : {
        "start" : 1649108544826,
        "stop" : 1649108544826,
        "duration" : 0
      }
    }, {
      "uid" : "3910b510ea48b407",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/11//#testresult/3910b510ea48b407",
      "status" : "passed",
      "time" : {
        "start" : 1649108210402,
        "stop" : 1649108211064,
        "duration" : 662
      }
    }, {
      "uid" : "43f664244e77d354",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/10//#testresult/43f664244e77d354",
      "status" : "passed",
      "time" : {
        "start" : 1649107864601,
        "stop" : 1649107865197,
        "duration" : 596
      }
    }, {
      "uid" : "985d8dac4423834b",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/9//#testresult/985d8dac4423834b",
      "status" : "passed",
      "time" : {
        "start" : 1649107514593,
        "stop" : 1649107515358,
        "duration" : 765
      }
    }, {
      "uid" : "90364937b93ecdd9",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/8//#testresult/90364937b93ecdd9",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x559891f03ad3 <unknown>\n#1 0x559891c63568 <unknown>\n#2 0x559891c86b37 <unknown>\n#3 0x559891c8223a <unknown>\n#4 0x559891cbd2ba <unknown>\n#5 0x559891cb74e3 <unknown>\n#6 0x559891c8cd1a <unknown>\n#7 0x559891c8de75 <unknown>\n#8 0x559891f31efd <unknown>\n#9 0x559891f4b19b <unknown>\n#10 0x559891f33c65 <unknown>\n#11 0x559891f4bec8 <unknown>\n#12 0x559891f27360 <unknown>\n#13 0x559891f67a68 <unknown>\n#14 0x559891f67be8 <unknown>\n#15 0x559891f817fd <unknown>\n#16 0x7fabf5d65609 <unknown>",
      "time" : {
        "start" : 1649107300763,
        "stop" : 1649107300763,
        "duration" : 0
      }
    }, {
      "uid" : "2eef5f1610afdbd1",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/7//#testresult/2eef5f1610afdbd1",
      "status" : "passed",
      "time" : {
        "start" : 1649106452311,
        "stop" : 1649106453097,
        "duration" : 786
      }
    }, {
      "uid" : "17ba9ec2f1b1e3fc",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/6//#testresult/17ba9ec2f1b1e3fc",
      "status" : "passed",
      "time" : {
        "start" : 1649106437452,
        "stop" : 1649106438131,
        "duration" : 679
      }
    }, {
      "uid" : "c5d7fcbade183839",
      "reportUrl" : "https://rupu1906.github.io/docker-selenium/5//#testresult/c5d7fcbade183839",
      "status" : "passed",
      "time" : {
        "start" : 1649106384201,
        "stop" : 1649106384935,
        "duration" : 734
      }
    } ]
  }
}